modules/seqexec/server/src/main/resources/Tcs.xml

```diff
@@ -1006,6 +1006,16 @@
     </Status>
     <Status name="tcsstate">
         <top>tcs</top>
+        <attribute name="crfollow">
+            <channel>crFollow</channel>
```

Comment:
crFollow.VAR, to be precise.

Reply:
[software@sbfgpidev1 ~]$ caget tcs:crFollow.VAL
tcs:crFollow.VAL               0